### PR TITLE
Adds `distance_to_camera` datatype in `TiledCamera`

### PR DESCRIPTION
**Pull request by pascal-roth**
# Description

Added the data type "distance_to_camera" in the `TiledCamera` class to be consistent with all other cameras. This data type is equal to the "depth" datatype.

## Type of change

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [x] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

**Comment by jtigue-bdai** [comment]
Do we know if the TiledCamera will get the distance_to_image_plane data type. We noticed last night what depth actually meant when converting to point cloud. The depth to point cloud function in IsaacLab assumes the distance_to_image_plane as the input data.

**Comment by kellyguo11** [comment]
This will be addressed as part of the update to the new tiled rendering API in isaac sim coming in the release. depth will become an alias to distance_to_image_plane

**Comment by pascal-roth** [comment]
as soon as the change of the new API becomes available, we can convert `depth_to_image_plane` in the entire code to `depth`. For now, this change here should be necessary to keep our interface similar over all camera implementations.

File: source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/camera/tiled_camera.py [hunk 1]
```diff
@@ -54,7 +56,7 @@ class TiledCamera(Camera):
     cfg: TiledCameraCfg
     """The configuration parameters."""
 
-    SUPPORTED_TYPES: set[str] = {"rgb", "depth"}
+    SUPPORTED_TYPES: set[str] = {"rgb", "distance_to_camera", "depth"}
     """The set of sensor types that are supported."""
```
Comment (jtigue-bdai) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/camera/tiled_camera.py]: Do we want to add a NOTE in the docstring that `depth` is an alias for `distance_to_camera`? Knowing full well we will have to change it in the release.

Comment (pascal-roth) [reply source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/camera/tiled_camera.py]: done

**Comment by jsmith-bdai** [comment]
> as soon as the change of the new API becomes available, we can convert `depth_to_image_plane` in the entire code to `depth`. For now, this change here should be necessary to keep our interface similar over all camera implementations.

Can you add a ticket to do this if it doesn't already exist?

**Comment by pascal-roth** [comment]
@jsmith-bdai out test will fail anyway as soon as the renaming takes place. I would avoid to have another issue open about that topic if this will anyway remind us, or what do you think about it?

**Review by Mayankm96 — APPROVED** [review]

**Pull request merged**